nntrainer/models/neuralnet.cpp

```diff
@@ -603,6 +603,44 @@ int NeuralNetwork::extendGraph(GraphType graph, std::string prefix) {
   return ML_ERROR_NONE;
 }
 
+std::vector<std::shared_ptr<Layer>>
+NeuralNetwork::getGraph(std::string input_layer, std::string output_layer) {
```

Comment:
How about `const std::string &` for the arguments?

Reply:
good suggestion - done.